project_event/models/mail_thread.py

```diff
@@ -25,14 +25,6 @@ def _message_track(self, tracked_fields, initial):
             track_visibility = getattr(self._fields[col_name], 'track_visibility', 'onchange')
             initial_value = initial[col_name]
             new_value = getattr(self, col_name)
```

Comment:
I do remember the spec for the message on the field employee_ids (The assign people). You are removing the message Assigned to and created by and letting the default odoo values. You are changing this to be handled in mail_tracking_value.py. But even if you are solving the case in which you don't want to format the message of field with the same name, what you are doing now is formating the messages of fields of the same type. So the problem is now more general.

Reply:
One possible solution is to check for the res.model to be project.task or project.project so the changes are applied only to these two models.